app/display.c

```diff
@@ -144,11 +144,14 @@ void display_init(void)
     clear_screen_region(ROW_FOOTER, 0, ROW_FOOTER, SCREEN_WIDTH - 1);
     prints(ROW_FOOTER, 0, " <ESC> Exit  <F1> Configuration  <Space> Scroll Lock");
     prints(ROW_FOOTER, 64, MT_VERSION "." GIT_HASH);
```

Comment:
MT_VERSION "." GIT_HASH is 4+1+7 = 12 characters log so arch-id has to start at column 76, otherwise it will overlap with GIT_HASH (or word "unknown)".  The code below shifts everything to 74, where we really need only more more character (at least for now) to fit "la64".

I think we just need to shift it one character left:
`prints(ROW_FOOTER, 63, MT_VERSION "." GIT_HASH);`

Then, arch-id should start at row 75.

Perhaps something like this would work even better:
```
    prints(ROW_FOOTER, 63, MT_VERSION "." GIT_HASH "."
#if defined (__x86_64__)
        "x64"
#elif defined (__i386__)
        "x32"
#elif defined (__loongarch_lp64)
        "la64"
#else
        "??"
#endif
    );
```

Note that I have pending changes for this file, if we don't want to change this patchset again, I can fix it later with a different PR.